core/bin/zksync_api/src/bin/dev_ticker_server.rs

```diff
@@ -0,0 +1,43 @@
+use actix_cors::Cors;
+use actix_web::{middleware, App, HttpServer};
+use providers::{
+    dev_liquidity_provider::config_liquidity_app, dev_price_provider::create_price_service,
+};
+use structopt::StructOpt;
+
+mod providers;
+
+#[derive(Debug, StructOpt, Clone, Copy)]
+struct FeeTickerOpts {
+    /// Activate "sloppy" mode.
+    ///
+    /// With the option, server will provide a random delay for requests
+    /// (60% of 0.1 delay, 30% of 0.1 - 1.0 delay, 10% of 5 seconds delay),
+    /// and will randomly return errors for 5% of requests.
+    #[structopt(long)]
+    sloppy: bool,
+}
+
+#[actix_web::main]
+async fn main() -> std::io::Result<()> {
+    let _vlog_guard = vlog::init();
+
+    let opts = FeeTickerOpts::from_args();
+    if opts.sloppy {
+        vlog::info!("Fee ticker server will run in a sloppy mode.");
+    }
+
+    HttpServer::new(move || {
+        App::new()
+            // .app_data(web::Data::new(storage.clone()))
```

Comment:
if we don't need the storage, I'd get rid of it

Reply:
done